AirCasting/CoreData/NSManagedObjectContext+Utils.swift

```diff
@@ -103,6 +103,14 @@ extension NSManagedObjectContext {
         new.setValue(uuid.rawValue, forKey: "uuid")
         return new
     }
+    
+    func checkForLocationlessSessions() throws -> Bool  {
```

Comment:
I don't quite dig this adding domain specific functions to `NSManagedObjectContext` using extensions. I think this should be handled solely by our tailored persistence/storage classes.

2nd thing here is that I don't think name of this functions matches return type:
- Hey, check for locationless sessions please
- Ok I checked that for you and `yes`.
:)

Maybe `anyLocationlessSessionsPresent` or something?